addons/binding/org.openhab.binding.allplay/src/main/java/org/openhab/binding/allplay/internal/AllPlayBindingProperties.java

```diff
@@ -17,25 +17,34 @@
  */
 public class AllPlayBindingProperties {
 
-    private final Integer rewindSkipTimeInSec;
-    private final Integer fastForwardSkipTimeInSec;
+    private final double rewindSkipTimeInSec;
```

Comment:
for reference: integer config parameters should not allow decimal input: https://github.com/eclipse/smarthome/issues/4115